assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -106,7 +127,7 @@ function Panel({
 
   // Persist when user collapses
   useEffect(() => {
-    if (!manuallyChanged) {
+    if (!manuallyChanged || !isPersisted) {
```

Comment:
Just for optics, I'd reverse these expressions. First and foremost, if this panel cannot ever be persisted, ignore everything else. Only secondly, check whether it has updated. It's a tiny thing, but I feel it reads more clearly.